.github/workflows/quick-jobs.yml

```diff
@@ -18,130 +18,135 @@ jobs:
   meta:
     name: Meta checks
     runs-on: ubuntu-latest
+    env:
+      cabal_build: >-
```

Comment:
Yes, I am aware that I am breaking the abstraction that the `Makefile` should offer, but I didn't see a way around it.
What I need here is a cache-key for some specific makefile goal(s), like e.g `spdx templates`.  Putting this computation into the Makefile e.g. as `spdx-templates-cache-hash` would also be a bit awkward, and, I would need to invoke `hashfiles` from the Makefile (although this should be solvable).

I think we should document which goals of the Makefiles are in which workflows, so that these are changed in sync.  E.g. the `lexer` goal was removed from the Makefile in #8980 but not from the workflow.

Reply:
Agreed. Future work then... Maybe a ticket.